ansible/roles_ocp_workloads/ocp4_workload_quarkus_super_heroes_demo/templates/argocd/argocd.yaml.j2

```diff
@@ -0,0 +1,15 @@
+apiVersion: argoproj.io/v1alpha1
+kind: ArgoCD
+metadata:
+  name: argocd
+  namespace: {{ t_project_name_argocd }}
+spec:
+  dex:
+    openShiftOAuth: true
+  server:
+    insecure: true
```

Comment:
Actually I found a better way. Remove insecure: true and make the termination reencrypt.

Reply:
Will do.